app/assets/stylesheets/taxs.scss

```diff
@@ -1,467 +1,930 @@
+    input[type="submit"].disabled:hover,
+    input[type="submit"][disabled="disabled"]:hover,
+    input[type="submit"][disabled]:hover {
+      cursor: default;
+      background-color: #00823b; }
+    *[role="button"].disabled:active, *[role="button"][disabled="disabled"]:active, *[role="button"][disabled]:active,
+    button.disabled:active,
+    button[disabled="disabled"]:active,
+    button[disabled]:active,
+    input[type="submit"].disabled:active,
+    input[type="submit"][disabled="disabled"]:active,
+    input[type="submit"][disabled]:active {
+      top: 0;
+      -webkit-box-shadow: 0 2px 0 #003518;
+      -moz-box-shadow: 0 2px 0 #003518;
+      box-shadow: 0 2px 0 #003518; }
+  *[role="button"]:link, *[role="button"]:hover, *[role="button"]:focus, *[role="button"]:visited,
+  button:link,
+  button:hover,
+  button:focus,
+  button:visited,
+  input[type="submit"]:link,
+  input[type="submit"]:hover,
+  input[type="submit"]:focus,
+  input[type="submit"]:visited {
+    color: #fff; }
+  *[role="button"]:before,
+  button:before,
+  input[type="submit"]:before {
+    content: "";
+    height: 110%;
+    width: 100%;
+    display: block;
+    background: transparent;
+    position: absolute;
+    top: 0;
+    left: 0; }
+  *[role="button"]:active:before,
+  button:active:before,
+  input[type="submit"]:active:before {
+    top: -10%;
+    height: 120%; }
+  *[role="button"][rel="external"]:after,
+  button[rel="external"]:after,
+  input[type="submit"][rel="external"]:after {
+    display: none;
+    content: none;
+    margin-left: 0;
+    margin-right: 0; }
+  *[role="button"]:focus,
+  button:focus,
+  input[type="submit"]:focus {
+    outline: 3px solid #ffbf47; }
+
 img + button,
 img + input[type="submit"] {
-  margin: $gutter 0;
-}
+  margin: 30px 0; }
+
 .label-inline {
-  @include inline-block;
-}
+  display: inline-block;
+  *display: inline;
+  zoom: 1; }
+
 .form-label-bold {
-  @include heading-24;
-  font-weight: 700;
-}
+  font-family: "nta", Arial, sans-serif;
+  font-size: 24px;
+  line-height: 1.25;
+  font-weight: 400;
+  text-transform: none;
+  padding-top: 9px;
+  padding-bottom: 6px;
+  font-weight: 700; }
+  @media (max-width: 640px) {
+    .form-label-bold {
+      font-size: 20px;
+      line-height: 1.2; } }
+  @media (min-width: 641px) {
+    .form-label-bold {
+      padding-top: 6px;
+      padding-bottom: 4px; } }
+
 input[type="text"],
 input[type="email"] {
   display: block;
   padding: 0.1em 0 0.1em 0.4em;
-  background-color: $grey-4;
-  border: 1px inset $grey-2;
+  background-color: #f8f8f8;
+  border: 1px inset #bfc1c3;
   font-size: 19px;
-  line-height: 1.2;
-}
+  line-height: 1.2; }
+
 input[type="submit"] {
-  display: block;
-}
+  display: block; }
+
 fieldset {
-  margin-bottom: $gutter / 2;
-}
+  margin-bottom: 15px; }
+
 .form-field {
-  margin-bottom: $gutter / 2;
-}
+  margin-bottom: 15px; }
 
 /*= Tables
 ----------------------------------------------*/
 article table {
```

Comment:
This tables section is really confusing. Does this service not have AF? There are so many rules that are unneeded or don't make sense. Can any of it be removed?

Reply:
That's why I deleted all those extra files SASS files. It seems this service had its own assets frontend but it seems useless because there was no sass compiler so all those styles weren't even getting included apart from taxs.css so I copied the css from taxs.css and pasted it in taxs.scss. I/someone will have to go through all the styles in taxs.scss and strip out anything thats not used. Perhaps it can be a new ticket?